homeassistant/components/samsungtv/media_player.py

```diff
@@ -105,7 +105,7 @@ async def stop_bridge(event):
         """Stop SamsungTV bridge connection."""
         bridge.stop()
 
-    hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, stop_bridge)
+    entry.async_on_unload(hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, stop_bridge))
```

Comment:
```suggestion
    config_entry.async_on_unload(hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, stop_bridge))
```

Looks like its named `config_entry` instead of `entry` above.